generate.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+generator_path="$(pwd)/airbyte-integrations/connector-templates/generator"
```

Comment:
we shouldn't put this in the root path of the repo as to not pollute it. Can we instead put it in `airbyte-integrations/connector-templates/generator/generate.sh` ?

Reply:
I can do this, but michel-tricot in his comment [here](https://github.com/airbytehq/airbyte/pull/3763#discussion_r643631803) asked whenever we can start the generator from the project root.

Reply:
I believe he meant the same thing based on the path he posted

Reply:
:white_check_mark: Done